wgsl/index.bs

```diff
@@ -1467,37 +1466,48 @@ following table:
 
 #### Structure Layout Rules ####  {#structure-layout-rules}
 
-Each structure member has a default size and alignment value. These values are
-used to calculate each member's byte offset from the start of the structure.
+Each structure |S| member M<sub>N</sub> has a size and alignment value, denoted
```

Comment:
the other place uses small "n"

Reply:
Fixed the `n` to be `N`, as there are many more `N`s.